modules/installation-launching-installer.adoc

```diff
@@ -206,6 +234,56 @@ link:https://cloud.redhat.com/openshift/install/pull-secret[Pull Secret] page on
 ifdef::openshift-origin[]
 This field is optional.
 endif::[]
+endif::rhv[]
+ifdef::rhv[]
+Respond to the installation program prompts.
+
+.. Optional: For `SSH Public Key`, select a password-less public key, such as `~/.ssh/id_rsa.pub`. This key authenticates connections with the new {product-title} cluster.
++
+[NOTE]
+====
+For production {product-title} clusters on which you want to perform installation debugging or disaster recovery on, select an SSH key that your `ssh-agent` process uses.
+====
+.. For `Platform`, select `ovirt`.
+.. For `Enter oVirt's API endpoint URL`, enter the URL of the {rh-virtualization} API using this format:
++
+----
+https://<engine-fqdn>/ovirt-engine/api <1>
+----
++
+<1> For `<engine-fqdn>`, specify the fully qualified domain name of the {rh-virtualization} environment.
++
+For example:
++
+----
+https://rhv-env.virtlab.example.com/ovirt-engine/api
+----
++
+.. For `Is the installed oVirt certificate trusted?`, enter `Yes` since you have already set up a CA certificate. Otherwise, enter `No`.
+.. For `oVirt's CA bundle`, if you entered `Yes` for the preceding question, copy the certificate from `/etc/pki/ca-trust/source/anchors/ca.pem` and paste it here. Then, press `Enter` twice. Otherwise, if you entered `No` for the preceding question, this question does not appear.
```

Comment:
I would say "certificate content" so that it's clear this is not a certificate path

Reply:
Fixed.